Sources/Sandbox.Game/Game/Entities/MyEntity.cs

```diff
@@ -477,17 +504,32 @@ public void CreateSync()
         public virtual void UpdateOnceBeforeFrame()
         {
             m_gameLogic.UpdateOnceBeforeFrame();
```

Comment:
Yes good eye, I will fix those.  UpdatingStopped wasn't ever called originally.  So I didn't include it.  Not sure if that was on purpose or not.